test-case/check-playback.sh

```diff
@@ -94,6 +104,17 @@ do
                     dmesg > $LOG_ROOT/aplay_error_${dev}_$i.txt
                     dloge "aplay on PCM $dev failed at $i/$loop_cnt."
                     exit 1
+                else
+                    #sleep random_min ~ ramdom_max seconds before running aplay, then aplay can be run under dsp active or suspended status randomly
+                    if [[ $(sof-dump-status.py --dsp_status 0) != "unsupported" ]] && [ ${OPT_VALUE_lst['R']} -eq 1 ]; then
+                        time=$(func_lib_get_random $random_min $random_max)
+                    else    
+                        dlogi "platform doesn't support runtime pm or random range is turned off, no need sleep time"
+                        time=$random_min
+                    fi
+                    sleep $time
+                    pm=`sof-dump-status.py --dsp_status 0`
+                    dlogi "Sleep: $time second(s), PM status is $pm"
```

Comment:
`no need sleep time`, but you set the time value to the `$random_min`

```suggestion
                    if [[ $(sof-dump-status.py --dsp_status 0) != "unsupported" ]] && [ ${OPT_VALUE_lst['R']} -eq 1 ]; then
                        time=$(func_lib_get_random $random_min $random_max)
                        sleep $time
                        dlogi "Sleep: $time second(s)"
                    else    
                        dlogi "platform doesn't support runtime pm or random range is turned off, no need sleep time"
                    fi
                    pm=`sof-dump-status.py --dsp_status 0`
                    dlogi "PM status is $pm"
```

Reply:
Thanks Bin, and still will keep sleep random_min for no need random delay scenarios.  And i will update the description for this.